docs/rules/useSelector-prefer-selectors.md

```diff
@@ -0,0 +1,59 @@
+#  Enforces that all useSelector properties use selector functions. (react-redux/useSelector-prefer-selectors)
```

Comment:
`Enforces that all useSelector properties use selector functions.` -- > `Enforces that all useSelector hooks use named selector functions` That  is how I understood the rule you are proposing. Please correct me if I am wrong